### PR TITLE
{lib}[GCCcore/10.2.0] FLAC v1.3.3, libogg v1.3.4

### PR DESCRIPTION
**Pull request by Micket**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by Micket** [comment]
Test report by @Micket
**FAILED**
Build succeeded for 0 out of 2 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/c96d724da7ffc74476b2e81b25b98f1c for a full test report.

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/0f9da254739b4b952613cba67fb2eead for a full test report.

**Comment by Micket** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3514.doduo.os - Linux RHEL 8.2, x86_64, AMD EPYC 7552 48-Core Processor (zen2), Python 3.6.8
See https://gist.github.com/e31ef7a4b55f876f54bf9261068a34c4 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3109.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/98c145df0990692d50dfe2f60113a0a2 for a full test report.

File: easybuild/easyconfigs/f/FLAC/FLAC-1.3.3-GCCcore-10.2.0.eb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+builddependencies = [('binutils', '2.35')]
+
+dependencies = [('libogg', '1.3.4')]
+
+configopts = '--enable-static --enable-shared'
+
+sanity_check_paths = {
+    'files': ['lib/libFLAC.a', 'lib/libFLAC++.a',
+              'lib/libFLAC.%s' % SHLIB_EXT, 'lib/libFLAC++.%s' % SHLIB_EXT,
+              'bin/flac'],
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLAC/FLAC-1.3.3-GCCcore-10.2.0.eb:24]: @Micket Ideally this is listed first (alphabetical order), and we have a `--help` sanity check command for it too?

File: easybuild/easyconfigs/f/FLAC/FLAC-1.3.3-GCCcore-10.2.0.eb [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    'dirs': ['include/FLAC', 'include/FLAC++'],
+}
```
Comment (boegel) [inline easybuild/easyconfigs/f/FLAC/FLAC-1.3.3-GCCcore-10.2.0.eb:26]: (I checked, this works)

```suggestion
sanity_check_paths = {
    'files': ['bin/flac', 'lib/libFLAC.a', 'lib/libFLAC++.a',
              'lib/libFLAC.%s' % SHLIB_EXT, 'lib/libFLAC++.%s' % SHLIB_EXT],
    'dirs': ['include/FLAC', 'include/FLAC++'],
}

sanity_check_commands = ["flac --help"]
```

**Review by boegel — CHANGES_REQUESTED** [review]

**Comment by Micket** [comment]
Test report by @Micket
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
vera-c1 - Linux centos linux 7.9.2009, x86_64, Intel Xeon Processor (Skylake), Python 2.7.5
See https://gist.github.com/b64e7cddd80b10331f06033b2b1d8545 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 2 out of 2 (2 easyconfigs in total)
node3109.skitty.os - Linux centos linux 7.9.2009, x86_64, Intel(R) Xeon(R) Gold 6140 CPU @ 2.30GHz, Python 3.6.8
See https://gist.github.com/b75481dad0b5cc4103c8101994d2b932 for a full test report.

**Review by boegel — APPROVED** [review]

**Pull request merged**